napari/_qt/qt_resources/_svg.py

```diff
@@ -88,7 +86,6 @@ def colored(
         return QColoredSVGIcon(self._svg, color, opacity)
 
     @staticmethod
-    @lru_cache
```

Comment:
I have removed the cache as the returned object is mutable and could be changed.